src/ipynb/spark_ml.ipynb

```diff
@@ -34,7 +34,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # Delete the Dataproc Interactive Session if session is existed
</pre><p>is existed -&gt; exists</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>